cartoframes/__init__.py

```diff
@@ -1,6 +1,18 @@
 from .context import CartoContext
 from .credentials import Credentials
-from .layer import BaseMap, QueryLayer, Layer
+from .layer import BaseMap, Layer
```

Comment:
these layers are for the carto.js/raster map implementation found in `CartoContext.map`

Reply:
Oh I see ;)

Reply:
But I guess it's another consideration for what to do with the name space